src/kava.ts

```diff
@@ -865,45 +863,48 @@ class Kava extends BasePlugin {
     return true;
   }
 
-  _logMissingParam(missingParam: string): void {
+  private _logMissingParam(missingParam: string): void {
     this.logger.warn(`Kava analytics block report because of missing param ${missingParam}`);
   }
 
-  static _getTimeDifferenceInSeconds(time: number): number {
+  private static _getTimeDifferenceInSeconds(time: number): number {
     return (Date.now() - time) / 1000.0;
   }
 
-  _updateTabModeInModel(hiddenAttr: string): void {
+  private _updateTabModeInModel(hiddenAttr: string): void {
     this._model.updateModel({
       // $FlowFixMe
       tabMode: this._isTabHidden(hiddenAttr) && !this.player.isInPictureInPicture() ? TabMode.TAB_NOT_FOCUSED : TabMode.TAB_FOCUSED
     });
   }
 
-  _isTabHidden(hiddenAttr: string): boolean {
+  private _isTabHidden(hiddenAttr: string): boolean {
     return document[hiddenAttr];
   }
 
-  _initTabMode(): void {
+  private _initTabMode(): void {
     let hiddenAttr: string;
     let visibilityChangeEventName: string;
     if (typeof document.hidden !== 'undefined') {
       // Opera 12.10 and Firefox 18 and later support
       hiddenAttr = 'hidden';
       visibilityChangeEventName = 'visibilitychange';
+      // @ts-expect-error - Property 'msHidden' does not exist on type 'Document'. Did you mean 'hidden'?
     } else if (typeof document.msHidden !== 'undefined') {
```

Comment:
is this for IE ?

Reply:
As a lesson from the previous refactor I do not change existing code